src/components/config-pages/header-buttons.tsx

```diff
@@ -0,0 +1,20 @@
+
+import Button from "@components/elements/button";
```

Comment:
For such a small bit, i wonder if it's just better to include the 2 lines into the page-header.tsx

Reply:
Let's do it!  😄